app/presenters/return-logs/base-return-logs.presenter.js

```diff
@@ -43,32 +43,6 @@ function formatMeterDetails(meter) {
   }
 }
 
-/**
- * Formats a return log's purposes into a string for display in the UI
- *
- * 99.999% of return logs the `metadata` field's `purposes` property is set. But there are 38 that we know of where
- * the return version information was deleted yet.
- *
- * The water-abstraction-import service still created return logs in WRLS, but they are missing key details like
- * purposes, points and the abstraction period.
- *
- * We don't want these few to block access to other returns for a licence. So, we have to handle them in code.
- *
- * @param {string[]} purposes - the list of purposes taken from the return log's metadata field
- *
- * @returns {string|null} The first purpose's alias, else tertiary description of the first purpose else an empty string
- * if purposes is empty
- */
-function formatPurpose(purposes) {
-  if (!purposes || purposes.length === 0) {
-    return ''
-  }
-
-  const [firstPurpose] = purposes
-
-  return firstPurpose.alias ? firstPurpose.alias : firstPurpose.tertiary.description
```

Comment:
From looking at this removed function it looks like we very much intentionally only displayed the first purpose. I'm not sure why that was, but am not really comfortable approving without knowing why we did it that way in the first place.

Reply:
This is the original ticket for info: https://eaflood.atlassian.net/browse/WATER-4444 Looks like we were just migrating an existing page. No mention of the purposes in there, guess it was just built as per the legacy page.

Reply:
Yes maybe just double check to make sure they are clear on the impact of the change.

Also I wouldn't necessarily label this as a "bug" as the system is working as intended. I'd say this is an "enhancement" as they are requesting a change to existing funtionality.